### DISCUSSION TITLE
Add "View Logs" (You can now view logs under the "Tools" button in GUI) and "Edit Config.yaml" button to GUI - Feature Request

### DISCUSSION THREAD

**Discussion by OverActiveBladderSystem** [Chia-Network/chia-blockchain-gui#2372] [Ideas]
How many times has a user sought help and the first question was, "Well what do the logs say?"

GUI users with limited experience may appreciate/benefit from a "Logs" button that opens the system file explorer to the directory Chia logs are stored within, to better help them self-diagnose or assist those trying to help.

Extension of idea, add buttons below to change LOG LEVEL between ERROR, WARN, INFO, DEBUG as well for those timid or inexperienced in updating the config.yaml so they may obtain more or less information as desired.

**Comment by OverActiveBladderSystem** [comment]
Expanding on the idea of a "view logs" button...

add an "edit config.yaml" with a plain text editor opening in the GUI.

This would help those troubleshooting with GUI users who need to inspect and/or edit their yaml, rather than walk the user through finding the file location and then making sure they are using a plain text editor (I believe Windows is removing the default plain text editor and opting for an RTF editor going forward, that may compound the users issues with yaml mistakes needing to init a new one).

**Comment by BrandtH22** [comment]
@OverActiveBladderSystem , note I transferred this enhancement request to the GUI repo for better visibility 👍

**Comment by OverActiveBladderSystem** [comment]
"View Logs" has been added to the GUI under the "Tools" section in the GUI, buttons on the left side of the screen.
